index.html

```diff
@@ -51,16 +48,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box-1">Box 1</div>
```

Comment:
Good thinking to change the class name so you can treat each of these boxes separately. What I would have done is this: 

```
<div class="box box-1">Box 1</div>
```

That way on the box class, you can write the code that dictates the height, width, etc...all the stuff that is the same amongst all the boxes. 

Then use the individual class name `box-1` to override the color on the box. 


So the CSS would look something like this :

```
.box {
    height: 100px;
    width: 100px;
    display: flex;
    justify-content: center;
    align-items: center;
    color: white;
}

.box-1 {
    background-color: teal;
}


```